IPython/core/magics/script.py

```diff
@@ -220,7 +220,7 @@ def in_thread(coro):
 
         async def _readchunk(stream):
             try:
-                return await stream.readuntil(b"\n")
+                return await stream.read(100)
```

Comment:
This will consume any up to 100 bytes, as soon as at least 1 byte is available. The 100 number is arbitrary, but my hope is it will roughly correspond to an average line length so that it should not change how small/frequent chunks are compared to before (but of course if someone streamed lines of 1000 characters, now those would be returned in 10 chunks instead of 1).

https://docs.python.org/3/library/asyncio-stream.html#asyncio.StreamReader.read